### ISSUE TITLE
extension does not load automatically based on file extension

### ISSUE THREAD

**Issue by ssbarnea** [GabLeRoux/ansi-colors-chrome-extension#8]
I tried to load `file://ansicolor.asc` or even `.txt` and the extension does nothing until I press the button.

As an user I expect to have it enabled by default for plain text files, or to have an extension list to which it would apply automatically.

**Comment by GabLeRoux** [comment]
Agreed, could also pass some user prefeferences for file extensions. Something we'll have to confirm is that we should not alter the file if user _downloads_ the file (ctrl+s/d+s), but I think it's ok by default.
